client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -65,7 +65,17 @@ export const JETPACK_SECURITY_CHECKLIST_TASKS: ChecklistTasks = {
 };
 
 export const JETPACK_PERFORMANCE_CHECKLIST_TASKS: ChecklistTasks = {
-	// No tasks for this checklist yet…
+	jetpack_lazy_images: {
+		title: translate( 'Lazy Load Images' ),
+		description: translate(
+			"Improve your site's speed by only loading images when visible on the screen."
+		),
+		getUrl: ( siteSlug, completed ) =>
```

Comment:
oops, can't use `T.URL` because this isn't a URL - it's a path. maybe we use that so often we need a new type. confusingly named function in any case

Reply:
[Path-absolute-URL strings](https://url.spec.whatwg.org/#path-absolute-url-string) are one example of valid URLs, right? 🤔 

It would seem fine to use `T.URL` here.